app/components/workspace/json-templates/end-user-workspace.json

```diff
@@ -0,0 +1,50 @@
+{
+    "id": "end-user.workspace",
+    "displayName": "Basic end-user workspace",
+    "description": "Default end-user workspace that hides a number of admin related features.",
+    "features": {
+      "pinned": false,
+        "job": {
+            "view": {
+                "statistics": false,
+                "tags": false,
+                "configuration": {
+                    "json": false
+                }
+            },
+            "action": {
+                "clone": false,
+                "exportAsJson": false,
+                "pin": false
+            }
+        },
+        "schedule": {
```

Comment:
feel like we could move all the view one step above. It doesn't really make sense to only disable view

```
{
 "job": {
   "statistics": true,
   "tags": true,
 }
}
```

Reply:
Action can stay where it is I reckon.

Reply:
After removing view, I need to rename the tags because 'tags' has been used for the summaryTags as well. 
<bl-tags *blFeatureOn="'pool.paper'" summaryTags [tags]="pool.tags" [editable]="true" [save]="updateTags" noTagsMessage="No tags."></bl-tags>